deployments/ocm-install/helmfile.yaml

```diff
@@ -0,0 +1,339 @@
+releases:
+  - name: ocis
+    chart: ../../charts/ocis
+    namespace: ocis
+    values:
+      - externalDomain: ocis.kube.owncloud.test
+      - ingress:
+          enabled: true
+          #ingressClassName: nginx
```

Comment:
we could remove this if we're relying on the default ingress class anyways.

Reply:
Ok will remove it, just ended up accidentally here as this is my make-it-work-with-k3d hack.

Reply:
Changed in https://github.com/owncloud/ocis-charts/pull/701/commits/470a949126f81ef62bad5fc9f65fd34bd03822dd

What about https://github.com/owncloud/ocis-charts/blob/470a949126f81ef62bad5fc9f65fd34bd03822dd/deployments/ocm-install/helmfile.yaml#L177-L178 ?

Should we adjust this as well?

Reply:
Completely removing might not be possible because the default seems to be `1m` and our default chunks size when doing TUS uploads seems to be 10MB (see the /ocs/v1.php/cloud/capabilities?format=json endpoint).

But we could decrease this value a lot by assuming there are only TUS enabled clients out there

I'm not sure which other limits might be applied for other ingress controllers...

In the end it boils down to:
- which ingress controllers are usable out of the box with our deployment examples (because we as oCIS chart devs are testing with them)
- which ingress controllers can be used at all (eg. Collabora limits to HAproxy and nginx-ingress)
- what is supported, if there is something like support for the oCIS chart...